### PR TITLE
[KEYCLOAK-8872] Keycloak 4.6.0.Final Script authenticator is missing

### PR DESCRIPTION
**Pull request by abstractj**

### CODE REVIEW

File: server_admin/topics/authentication/flows.adoc [hunk 1]
```diff
@@ -53,7 +53,14 @@ Executions can be used
 
 .Script Authenticator
 A _script_ authenticator allows to define custom authentication logic via JavaScript.
-Custom authenticators. Authentication scripts must at least provide one of the following functions:
+Custom authenticators. In order to make use of this feature, it must be explicitly enabled:
+[source]
+----
+bin/standalone.sh|bat -Dkeycloak.profile.feature.scripts=enabled
+----
+For more information see link:{installguide_profile_link}[{installguide_profile_name}] section.
```
Comment (matthewhelmke) [inline server_admin/topics/authentication/flows.adoc]: You need a comma and a "the" here:
For more information, see the...

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Just a couple minor things to request.

**Comment by abstractj** [comment]
@matthewhelmke thank you! Updated.

**Review by matthewhelmke — APPROVED** [review]
Thank you.

**Review by mposolda — APPROVED** [review]

**Comment by mposolda** [comment]
@matthewhelmke Could this be merged? I am not merging as it's assigned to you.

**Comment by matthewhelmke** [comment]
@mposolda yes, I'll merge it now.

**Pull request merged**